scripts/clean-shrinkwrap.js

```diff
@@ -25,7 +25,7 @@ const EXIT_CODES = require('../lib/shared/exit-codes');
 const SHRINKWRAP_PATH = path.join(__dirname, '..', 'npm-shrinkwrap.json');
 
 try {
-  console.log(childProcess.execSync('npm shrinkwrap', {
+  console.log(childProcess.execSync('npm shrinkwrap --dev', {
     cwd: path.dirname(SHRINKWRAP_PATH)
   }));
```

Comment:
Regarding this scripts, I think the complexity of this script lies in the fact that it call `npm shrinkwrap` itself, when its not really its responsibility. Now that we're shrinkwrapping development dependencies, we shouldn't need to call `npm shrinkwrap` ourselves (npm will do it automatically for us whenever we do `npm install`, `npm uninstall`, etc).

I edited the script to only scans the current shrinkwrap file and remove the optional dependencies.